File: src/main/java/org/intellij/sdk/language/angelscript/AngelScriptLexerAdapter.java
package org.intellij.sdk.language.angelscript;

import com.intellij.lexer.FlexAdapter;
import org.intellij.sdk.language.angelscript._AngelScriptLexer;

public class AngelScriptLexerAdapter extends FlexAdapter {
	public AngelScriptLexerAdapter() {
		super( new _AngelScriptLexer( null ) );
	}
}

File: src/main/kotlin/com/enderzombi102/css/lang/angelscript/AngelScriptLexerAdapter.kt
package com.enderzombi102.css.lang.angelscript

import com.intellij.lexer.FlexAdapter
import com.enderzombi102.css.lang.angelscript.grammar._AngelScriptLexer

class AngelScriptLexerAdapter : FlexAdapter( _AngelScriptLexer(null) )